spring-boot-testing/src/main/java/com/baeldung/testlog/TestLog.java

```diff
@@ -0,0 +1,22 @@
+package com.baeldung.testlog;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TestLog {
+
+	Logger LOG = LoggerFactory.getLogger(this.getClass());
```

Comment:
Generally speaking a logger would be `private static final` - especially if it has an uppercase name.